includes/class-theme-manager.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Newspack theme manager
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+use \WP_Error;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class for managing installation/activation of Newspack theme and child themes.
+ */
+class Theme_Manager {
+
+	/**
+	 * Array of valid Newspack theme slugs.
+	 *
+	 * @var array
+	 */
+	public static $theme_slugs = [
```

Comment:
@laurelfulford do you have names for the two remaining child themes yet (Style Packs 3 & 5)?

Reply:
Sorry for the late reply! They are, respectively, Newspack Katharine and Newspack Joseph.